utils/run_test.sh

```diff
@@ -98,6 +98,13 @@ if [ -d "/mnt/daos" ]; then
         export DAOS_IO_BYPASS=pm_snap
         run_test "${SL_PREFIX}/bin/vos_tests" -A 50
         unset DAOS_IO_BYPASS
+        COMP="UTEST_common"
+        run_test src/common/tests/btree.sh perf -s 20000
+        run_test src/common/tests/btree.sh perf direct -s 20000
+        run_test src/common/tests/btree.sh perf ukey -s 20000
+        run_test src/common/tests/btree.sh dyn perf -s 20000
+        run_test src/common/tests/btree.sh dyn perf ukey -s 20000
+        BTREE_SIZE=20000
```

Comment:
Might be better to set BTREE_SIZE first and use the variable when calling the tests